pkg/devicetwin/dtmanager/membership_test.go

```diff
@@ -0,0 +1,355 @@
+/*
+Copyright 2019 The KubeEdge Authors.
```

Comment:
nit: Lets not remove blank lines from the license

Reply:
Modified.

Reply:
Just this thing left, Rest LGTM

Reply:
Modified.